alibi_detect/saving/tests/test_saving.py

```diff
@@ -553,7 +554,7 @@ def test_save_contextmmddrift(data, kernel, backend, tmp_path, seed):  # noqa: F
     assert cd_load._detector.n_permutations == N_PERMUTATIONS
     assert cd_load._detector.p_val == P_VAL
     assert isinstance(cd_load._detector.preprocess_fn, Callable)
-    assert cd_load._detector.preprocess_fn.func.__name__ == 'preprocess_simple'
+    assert cd_load._detector.preprocess_fn.__name__ == 'preprocess_simple'
```

Comment:
Change required since we no longer wrap the function in a `partial` when there are no kwargs.